.github/workflows/node.test.yaml

```diff
@@ -6,6 +6,8 @@ on:
       - "master"
     paths:
       - "node/**"
+    paths-ignore:
+      - "node/pkg/logscribe/**"
```

Comment:
unfortunately, paths and paths-ignore can't be used together 😞

(reference)[https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore]

Reply:
I see. Then I wonder why do want to have a separate `logscribe` workflow if the `logscribe` tests will be triggered from `node` workflow as well.

Reply:
this seems valid, can you try this?
https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#example-including-and-excluding-paths

Reply:
good finding, I think that should work, will test it out